examples/patterns/pagination.html

```diff
@@ -5,11 +5,11 @@
 ---
 
 <footer class="p-pagination">
-  <a class="p-pagination__link" href="#previous">
+  <a class="p-pagination__link p-pagination__previous" href="#previous">
```

Comment:
Can you make these a modification instead of a new class. For example: `p-pagination__link--previous`, `p-pagination__link--next` and a default `p-pagination__link`